sensu/server/_handler_pagerduty.sls

```diff
@@ -0,0 +1,42 @@
+
+gem_sensu_pagerduty:
```

Comment:
Do we really want to start messing up package-based systems with gem install / pip install etc.?
I would prefer having an option which installation method to use or force package.